### DISCUSSION TITLE
rdb

### DISCUSSION THREAD

**Discussion by Smolsada** [dragonflydb/dragonfly#2185] [Q&A] [answered]
Hello!
I want to use redis persistence in k8s.
I used [this](https://github.com/dragonflydb/dragonfly/blob/c4cae58fc8ae0f0198bdc2c9f4cbd46bc0ef9445/contrib/charts/dragonfly/values.yaml#L89) setting, but encountered that the data is not saved automatically.

Please tell me how can I use these redis settings to save data automatically?

```yaml
rdb: true
save:
  - "900 1"
  - "300 10"
  - "60 10000"
appendonly: true
---

**Comment by dranikpg** [comment, marked as answer]
You can use the snapshot_cron flag to save it periodically, we don't support redis frequency intervals

**Reply by Smolsada** [reply]
Understood. Thank you!
